### ISSUE TITLE
Deprecate this crate in favor of the original `meshopt`

### ISSUE THREAD

**Issue by oisyn** [Uriopass/meshopt2#2]
Now that we have maintainer access to the original `meshopt`, shouldn't we just clear out this fork or at least change the readme to refer back to the original `meshopt2`?

**Comment by Uriopass** [comment]
Absolutely! Maybe I can contact the crates.io maintainers to remove the crate entirely.
